apps/lockfile-explorer-web/src/containers/LogoPanel/index.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import styles from './styles.scss';
+
+export const LogoPanel = (): JSX.Element => {
+  const appPackageVersion: string = '0.0.0';
```

Comment:
This should be automatically kept in sync with the `apps/lockfile-explorer/package.json` sibling project, but I'm still thinking about the best way to do that.

@iclanton @elliot-nelson